src/popup/utils/helper.js

```diff
@@ -181,3 +182,13 @@ export const isValidURL = (url) => {
   const pattern = new RegExp(/((http(s)?:\/\/)?(localhost|127.0.0.1)((:)?[\0-9]{4})?\/?)/, 'i');
   return isURL(url) || !!pattern.test(url);
 };
+
+const hasValidationKey = (error, expectedKey) => (
+  error.validation
+  && error.validation.some(({ key }) => expectedKey === key)
+);
+
+export const isInsufficientBalanceError = (error) => (
+  error instanceof InvalidTxError
+  && hasValidationKey(error, 'InsufficientBalance')
+);
```

Comment:
Please avoid adding new utils to non-TS files. There is pending PR for moving all stuff from this file into the utils.ts file.